src/fidesctl/cli/utils.py

```diff
@@ -40,10 +40,11 @@ def check_server(cli_version: str, server_url: str) -> None:
         raise SystemExit(1)
 
     server_version = health_response.json()["version"]
-    if str(server_version) == str(cli_version):
-        echo_green(
-            "Server is reachable and the client/server application versions match."
-        )
+    if str(server_version).removesuffix(".dirty") == str(cli_version):
```

Comment:
Good catch on the Python 3.9 requirement. We can avoid that with:
```suggestion
    if str(server_version).replace(".dirty", "", 1) == str(cli_version):
```